### ISSUE TITLE
[ISSUE]: PS2HDMI Fix for PS2 HDMI dongles Black Screen

### ISSUE THREAD

**Issue by PhilRoll** [DKWDRV/DKWDRV#61]
### Checks

- [X] I have checked existing [__DKWDRV issues__](https://github.com/DKWDRV/DKWDRV/issues) for duplicates and found none.
- [X] I am using [latest version](https://github.com/DKWDRV/DKWDRV/releases)
- [X] CD/USB - I have checked my game redump hashes/verified it is a good dump.
- [ ] USB - I have merged the bin tracks (if game has several ones).
- [ ] USB - I have made sure that game is defragemented. You can boot even fragemented games but 99% chance of random failure.

### Context and extra information for the game (if applicable)

all CD games affectect 

### Describe the issue(attach photos of the two debug screens if you get them)

I wanted to use DKWDRV  only to run PS1 cd games with PS2 HDMI converter.
I tried to turn on the fix, save config, load cd disk and then RUN, but black screen appear.

My question is:
even if my ps2 is not among the supported models (scph 39004), shouldn't the fix work?

PS: the console has a modchip installed (Matrix MXL2).
i OBVIOUSLY deactivated the modchip before starting DKWDRV. Tried also to reset the HDMI adapter but nothing happens

### Console model and type

SCPH-39004

### Console details

Modchip Matrix MXL2, FreeMCBoot 1.96

### DKWDRV

DKWDRV 1.7.6

### DKWDRV Options

PS2HDMI Fix for PS2 HDMI dongles = 1

### Tell us what method are you trying to run the game from. CD or USB.

CD

**Comment by DKWDRV** [comment]
@PhilRoll all PS2 models support HDMI fix. Have you rechecked in menu if fix is enabled after saving?

At minimum the PS1 logo would almost always show up no matter what. Can you hear the disc spinning? 

What happens if you use composite cables?

Some modchips can not be fully deactivated (could be mistaken here).

**Comment by DKWDRV** [comment]
What specific game are you trying? Or several ones?

What happens if you use PS1VmodeNeg or GSM PS1?

**Comment by DKWDRV** [comment]
I think the driver simply crashes on your console (as confirmed by no spinning disc at all). Since you have no debug output I can not think of any particular reason then.
Anything special with your PS2? Extra hardware accessory? 

Some modchips still keep ROM patching. Have you dumped your BIOS with and without modchip and confirm the hashes with correct known BIOS dumps for that model?

How are you launching DKWDRV?

**Comment by PhilRoll** [comment]
> I think the driver simply crashes on your console (as confirmed by no spinning disc at all). Since you have no debug output I can not think of any particular reason then. Anything special with your PS2? Extra hardware accessory?
> 
> Some modchips still keep ROM patching. Have you dumped your BIOS with and without modchip and confirm the hashes with correct known BIOS dumps for that model?
> 
> How are you launching DKWDRV?

i use ps2 network adapter with 2.5 hdd. no sorry, I purchased the console already modchipped, i have no dump.
i launch elf from USB (with ulaunchElf)

**Comment by DKWDRV** [comment]
You can make dumps with PS2 Ident. Compare with modchip on and modchip off. And compare with correct hash for your model.

And if you remove the network adapter for testing?

**Comment by DKWDRV** [comment]
Can you share your NVRAM (privately)?
Might be related to https://github.com/DKWDRV/DKWDRV/issues/74

**Comment by DKWDRV** [comment]
@PhilRoll can you try setting video mode to force NTSC or PAL and test?

**Comment by PhilRoll** [comment]
@DKWDRV these are my tests:
- remove the PS2 Network adapter -> black screen after ps logo if i use "Launch Original PS1DRV", total black screen and no disc spin if i use "RUN DISC"
- force PAL/NTSC with GSM -> works with "Launch Original PS1DRV", total black screen and no disc spin if i use "RUN DISC"

Sorry i don't have time to learn how to use PS2 Ident/dump NVRAM. 

(PS: what is the correct form to start the games between "Launch Original PS1DRV" and "RUN DISC"? )

**Comment by DKWDRV** [comment]
Can you try force mode to NTSC or PAL with DKWDRV (not GSM) ?

"Launch Original PS1DRV" - Same as running vanilla default driver software.
"RUN DISC" - Uses DKWDRV

**Comment by DKWDRV** [comment]
Can you try with 1.7.6b and AUTO video mode in settings?

**Comment by PhilRoll** [comment]
> Can you try with 1.7.6b and AUTO video mode in settings?

i tried, still have black screen

**Comment by DKWDRV** [comment]
Did you try to force any other mode at least? NTSC or PAL from the options menu?

I have no idea what else can be causing this.

**Comment by PhilRoll** [comment]
> Did you try to force any other mode at least? NTSC or PAL from the options menu?
> 
> I have no idea what else can be causing this.

yes, i also tried to force pal/ntsc mode, nothing to do... still black screen after i select run disc

**Comment by DKWDRV** [comment]
No idea what the problem can be in your case.

**Comment by dskaufman1** [comment]
Could the issue be because some PS1 games display on 240p (or 270 pal) and the PS2 HDMI adapter can't display that resolution?  For instance I can't run ICO using my PS2 HDMI adapter because it outputs lower resolution than supported.

**Comment by PhilRoll** [comment]
> [@PhilRoll](https://github.com/PhilRoll) still insist is your modchip doing this. Can you dump your BIOS with modchip disabled and compare it with redump BIOS?

I was able to make the dumps. I made 3 versions of the dump:

- modchip disabled
- modchip in ps2 mode
- modchip in ps1 mode

I calculated the hash of the various files and these are the results:
SCPH-39004.mec✅ 
SCPH-39004.nvm✅ 
SCPH-39004.rom0 ❌ (differences)
SCPH-39004.rom1 ❌ (differences)
SCPH-39004.rom2 ✅

**Comment by DKWDRV** [comment]
@PhilRoll that means the chip is not properly disabled or it only disabled mechacon patching! If you want you can extract BIOS files and see what files are being patched by the BIOS. Most likely a couple of bytes in PS1DRV.

**Comment by PhilRoll** [comment]
> [@PhilRoll](https://github.com/PhilRoll) try latest version, if no more updates the issue will be closed.

the problems still persist.. modchip disabled, PS1 disc correctly recognized, screen always black after run

**Comment by DKWDRV** [comment]
@PhilRoll can you try latest version?
How can we see what the modchip has patched in the bios? Can you diff bios files?

**Comment by PhilRoll** [comment]
same issue...

hash with modchip OFF:
a3e4c2f5c5c1d4e79c06de43a10328d4bde74d7993570b8963806ee8ef8a4b0d  SCPH-39004.mec
2989f0372c70e3e656ff16eaf20118861f040bb75fb84d89e28b0ceeae5dbc11  SCPH-39004.nvm
5b7cfc4dfdcf556d70ea975b754da6ec26542f6a0e3dfa43a6fcef2902e8f60d  SCPH-39004.rom0
4ee0825757bd044de93dbecdd6b99f03d59759c65425864f78c92dcb5e1d967d  SCPH-39004.rom1
bea26a3576ae6bc1ef9c8d768c8f4687e1532ff0a29811f37e04de01c2a4c1ae  SCPH-39004.rom2

hash with modchip ON 
a3e4c2f5c5c1d4e79c06de43a10328d4bde74d7993570b8963806ee8ef8a4b0d  SCPH-39004.mec
2989f0372c70e3e656ff16eaf20118861f040bb75fb84d89e28b0ceeae5dbc11  SCPH-39004.nvm
7068c8e62038820a8096ad230cde811376ab6e42691de37fd4918837b835c431  SCPH-39004.rom0
923a48b8dba7d6e1d9343c471893b30deb97cc8cad3444a2f7a57dd1a8d6f97b  SCPH-39004.rom1
bea26a3576ae6bc1ef9c8d768c8f4687e1532ff0a29811f37e04de01c2a4c1ae  SCPH-39004.rom2

**Comment by DKWDRV** [comment]
> same issue...
> 
> hash with modchip OFF: a3e4c2f5c5c1d4e79c06de43a10328d4bde74d7993570b8963806ee8ef8a4b0d SCPH-39004.mec 2989f0372c70e3e656ff16eaf20118861f040bb75fb84d89e28b0ceeae5dbc11 SCPH-39004.nvm 5b7cfc4dfdcf556d70ea975b754da6ec26542f6a0e3dfa43a6fcef2902e8f60d SCPH-39004.rom0 4ee0825757bd044de93dbecdd6b99f03d59759c65425864f78c92dcb5e1d967d SCPH-39004.rom1 bea26a3576ae6bc1ef9c8d768c8f4687e1532ff0a29811f37e04de01c2a4c1ae SCPH-39004.rom2
> 
> hash with modchip ON a3e4c2f5c5c1d4e79c06de43a10328d4bde74d7993570b8963806ee8ef8a4b0d SCPH-39004.mec 2989f0372c70e3e656ff16eaf20118861f040bb75fb84d89e28b0ceeae5dbc11 SCPH-39004.nvm 7068c8e62038820a8096ad230cde811376ab6e42691de37fd4918837b835c431 SCPH-39004.rom0 923a48b8dba7d6e1d9343c471893b30deb97cc8cad3444a2f7a57dd1a8d6f97b SCPH-39004.rom1 bea26a3576ae6bc1ef9c8d768c8f4687e1532ff0a29811f37e04de01c2a4c1ae SCPH-39004.rom2

Need the two dumps to compare what has changed.

I think the loading address impacts this chip. Try compiling vmodeneg in different loading address and the same will likely happen.

**Comment by DKWDRV** [comment]
What disc type is shown in INFO after scanning disc?

**Comment by PhilRoll** [comment]
> What disc type is shown in INFO after scanning disc?

Disc Type: 0x10
Console Region and Disc License: E

**Comment by PhilRoll** [comment]
> > same issue...
> > hash with modchip OFF: a3e4c2f5c5c1d4e79c06de43a10328d4bde74d7993570b8963806ee8ef8a4b0d SCPH-39004.mec 2989f0372c70e3e656ff16eaf20118861f040bb75fb84d89e28b0ceeae5dbc11 SCPH-39004.nvm 5b7cfc4dfdcf556d70ea975b754da6ec26542f6a0e3dfa43a6fcef2902e8f60d SCPH-39004.rom0 4ee0825757bd044de93dbecdd6b99f03d59759c65425864f78c92dcb5e1d967d SCPH-39004.rom1 bea26a3576ae6bc1ef9c8d768c8f4687e1532ff0a29811f37e04de01c2a4c1ae SCPH-39004.rom2
> > hash with modchip ON a3e4c2f5c5c1d4e79c06de43a10328d4bde74d7993570b8963806ee8ef8a4b0d SCPH-39004.mec 2989f0372c70e3e656ff16eaf20118861f040bb75fb84d89e28b0ceeae5dbc11 SCPH-39004.nvm 7068c8e62038820a8096ad230cde811376ab6e42691de37fd4918837b835c431 SCPH-39004.rom0 923a48b8dba7d6e1d9343c471893b30deb97cc8cad3444a2f7a57dd1a8d6f97b SCPH-39004.rom1 bea26a3576ae6bc1ef9c8d768c8f4687e1532ff0a29811f37e04de01c2a4c1ae SCPH-39004.rom2
> 
> Need the two dumps to compare what has changed.
> 
> I think the loading address impacts this chip. Try compiling vmodeneg in different loading address and the same will likely happen.

I think we can't share the BIOS dump here, I think it's best to avoid sharing these files here.

For vmodeneg: at the moment, I don't have the tools or the time to compile it, but if I get a chance, I'll try.
Any advice on how to choose free addresses?

**Comment by neoconzer** [comment]
First, I think we can basically rule out an HDMI adapter issue. Second, I also suspect that the issue is caused by the modchip, because the original poster encountered this issue very early on, but I didn't encounter this issue until 1.7.6f, https://github.com/DKWDRV/DKWDRV/issues/116, Furthermore, in case https://github.com/DKWDRV/DKWDRV/issues/74, the user didn't have the modchip installed. So far, this is quite a strange issue.

**Comment by DKWDRV** [comment]
@PhilRoll  we need a way to see what them, even if temporary.

@neoconzer yes, #74 is confusing report

**Comment by DKWDRV** [comment]
@PhilRoll this is the last time I look into such issue. One last test on the recent version?

**Comment by PhilRoll** [comment]
I'm sorry, but unfortunately I still have the same problem.
I've tried everything I can. It doesn't work.

Chipmod disabled, disk recognized (scan disk ok), but as soon as I click "RUN" I get a black screen (the disk isn't spinning, it doesn't make any noise).

**Comment by DKWDRV** [comment]
You have another PS2 to test?
I am fairly sure it is still the modchip issue. Likely doing in fly patching when it detects rom0:PS1DRV somehow. Can you extract and compare which file of rom0 has changed between chip and vanilla bios dump?

**Comment by DKWDRV** [comment]
@PhilRoll try with the latest version. This time it will boot!

**Comment by PhilRoll** [comment]
@DKWDRV no :'(  it doens't...

All I could do was dump my own with the chip both on and off (see my earlier posts above). I don't have another SCPH-39004 PS2 to test the vanilla BIOS versus the chip BIOS dump.

I don't know what to say, other than thank you as always for all the hard work you've put in.


PS: If you want, you can take a look at my homebrew (a fork of GSM) that reads PS1 games and runs them on screens with component/HDMI ( [https://github.com/PhilRoll/gsm](url) ).
I wrote it a while back, so I don’t really remember much about it anymore, but the source code is public.
Maybe you can get something out of it—I’m not sure.

**Comment by DKWDRV** [comment]
@PhilRoll can you try with different Component fix options?
Why not unpack your two bios dump chips and tell us which file of the rom changes?

**Comment by PhilRoll** [comment]
> [@PhilRoll](https://github.com/PhilRoll) can you try with different Component fix options? Why not unpack your two bios dump chips and tell us which file of the rom changes?

@DKWDRV Yes, i tried component/ps2hdmi fix disabled, enabled and progressive mode (wip). 
The problem is that the disc isn't spinning at all.
“Scan Disc” works, and I can see the data correctly under “Info”.
But as soon as I press “RUN DISC,” the screen goes black (or there's no video output if I disable the component fix)
And, most important thing,  the disc isn't spinning, in any video mode after pressing run.


I dumped and unpacked the BIOS for all **3 states** of the modchip (Matrix MXL2):

- Chipmod **OFF**
- Chipmod **ON – PS2 mode**
- Chipmod **ON – PSX mode**

Console: **SCPH-39004**

| File | OFF | PS2 mode | PSX mode | Result |
|------|-----|----------|----------|:------:|
| rom0 | `0d2228e6fd4fb639c9c39d077a9ec10c` | `cc9f165880b52706c00abe0cdb1d6d4b` | `49ae2b543d6709424dc4413f8a83f0ba` | ❌ all different |
| rom1 | `f3dfc19904e169ba1c77e6e28dc63acc` | `a90245347069ce6bfb17886017e0f251` | `f3dfc19904e169ba1c77e6e28dc63acc` | ❌ PS2 mode differs |
| rom2 | `408579dfa40d6f9a0bd87cf1d8bc6a34` | same | same | ✅ identical |
| nvm  | `5ed66bcef0550b55bc91d0afde6f728a` | same | same | ✅ identical |
| mec  | `50500a02816976628774affb8632c6cc` | same | same | ✅ identical |



Even though `rom0` and `rom1` differ as whole files, unpacking each ROM into its individual modules shows the actual changes are minimal:

**rom0** (92 modules) — only **2 modules** differ:

| Module | # | Differs in | Notes |
|--------|---|------------|-------|
| `ROMGSCRT` | 54 | **PSX mode** only | OFF and PS2 mode are identical to each other |
| `PS2LOGO`  | 88 | **PS2 mode** only | OFF and PSX mode are identical to each other |

All other 90 modules in rom0 are byte-identical across all 3 states.

**rom1** (15 modules) — **0 modules** differ, despite the file MD5 changing in PS2 mode. That means the difference lives in a region of rom1 that isn't covered by the module table (likely boot/config data specific to the modchip's IOP patching, not extracted as a named module).

**rom2, nvm, mec** — fully identical in all 3 states, both at file and module level.

**Comment by DKWDRV** [comment]
@PhilRoll one more test with version 1.7.6k which got released! Maybe that will help!

**Comment by PhilRoll** [comment]
> [@PhilRoll](https://github.com/PhilRoll) one more test with version 1.7.6k which got released! Maybe that will help!

nothing, I've tried every possible test, and it still doesn't work :'( .

**Comment by DKWDRV** [comment]
> > [@PhilRoll](https://github.com/PhilRoll) one more test with version 1.7.6k which got released! Maybe that will help!
> 
> nothing, I've tried every possible test, and it still doesn't work :'( .

I really don't know what the problem with your cursed console is.

**Comment by PhilRoll** [comment]
> > > [@PhilRoll](https://github.com/PhilRoll) one more test with version 1.7.6k which got released! Maybe that will help!
> > 
> > 
> > nothing, I've tried every possible test, and it still doesn't work :'( .
> 
> I really don't know what the problem with your cursed console is.

dunno, I tried all the possible combinations:
for both modes chipmod off and chipmod in PS1 mode (chipmod in PS2 mode no because otherwise the disc scan won't work):
- without component fix
- with component fix (both enabled and progressive scan)
- force PAL
- force NTSC
- GPU GP1 Reset CMD Homebrew Fix: 0 and 1
- Crackto Distorted Screen Fix: 0 and 1

I ran all these tests both with and without the PS2 network adapter.

I don't know what else I can test...

**Comment by DKWDRV** [comment]
How bold are you to physically disable the modchip?

**Comment by PhilRoll** [comment]
> How bold are you to physically disable the modchip?

I don't know how to do it; otherwise, I would have already tried unsoldering it and resoldering it. I'm sorry.

**Comment by DKWDRV** [comment]
Let's wait for neocozer test on another issue, if it works for all of the rest it's possible it is still a modchip issue.

**Comment by PhilRoll** [comment]
@DKWDRV I put together a more thorough BIOS dump comparison (all 3 modchip states, file-level + module-level MD5s) — see the attached report.
I hope you find this useful.

# BIOS Diff Report — SCPH-39004

Binary comparison (offsets + hex diff) between the 3 dumps: **off**, **ps2-mode**, **psx-mode**.


## MD5 Summary

| File | modchip-off | modchip-ps2-mode | modchip-psx-mode | 
|------|------|------|------|
| SCPH-39004.mec | `50500a02816976628774affb8632c6cc` | `50500a02816976628774affb8632c6cc` | `50500a02816976628774affb8632c6cc` |
| SCPH-39004.nvm | `5ed66bcef0550b55bc91d0afde6f728a` | `5ed66bcef0550b55bc91d0afde6f728a` | `5ed66bcef0550b55bc91d0afde6f728a` |
| SCPH-39004.rom0 | `0d2228e6fd4fb639c9c39d077a9ec10c` | `cc9f165880b52706c00abe0cdb1d6d4b` | `49ae2b543d6709424dc4413f8a83f0ba` |
| SCPH-39004.rom1 | `f3dfc19904e169ba1c77e6e28dc63acc` | `a90245347069ce6bfb17886017e0f251` | `f3dfc19904e169ba1c77e6e28dc63acc` |
| SCPH-39004.rom2 | `408579dfa40d6f9a0bd87cf1d8bc6a34` | `408579dfa40d6f9a0bd87cf1d8bc6a34` | `408579dfa40d6f9a0bd87cf1d8bc6a34` |

---

## 🔹 modchip-off vs modchip-ps2-mode

### SCPH-39004.mec

✅ Identical — MD5 `50500a02816976628774affb8632c6cc`

### SCPH-39004.nvm

✅ Identical — MD5 `5ed66bcef0550b55bc91d0afde6f728a`

### SCPH-39004.rom0

❌ Different — `0d2228e6fd4fb639c9c39d077a9ec10c` vs `cc9f165880b52706c00abe0cdb1d6d4b`

- Total differing bytes: **75**

#### Region 1 — offset ~0x3054C4 (3167428 decimal)

```diff
--- diff-report/modchip-off_vs_modchip-ps2-mode_SCPH-39004.rom0_region1_A.hex	2026-07-05 22:43:29.996065714 +0200
+++ diff-report/modchip-off_vs_modchip-ps2-mode_SCPH-39004.rom0_region1_B.hex	2026-07-05 22:43:29.997009720 +0200
@@ -2,12 +2,12 @@
 003054d4: 0000 0000 0000 0000 faff 6014 ffff 6324  ..........`...c$
 003054e4: f4ff 4014 2d20 0000 2d28 0000 202d 040c  ..@.- ..-(.. -..
 003054f4: 2d30 0000 d080 80af 1e08 040c 0000 0000  -0..............
-00305504: 2d18 4000 0500 6010 0200 0224 1300 6210  -.@...`....$..b.
-00305514: cc80 878f 2200 0010 0000 0000 cc80 878f  ...."...........
-00305524: 2d30 0000 2d28 2002 0000 a28c 0100 c624  -0..-( ........$
-00305534: d080 838f 0400 a524 0018 c42c 2118 6200  .......$...,!.b.
-00305544: d080 83af f9ff 8054 0000 a28c d080 838f  .......T........
-00305554: db62 023c 1000 0010 661e 4234 2d30 0000  .b.<....f.B4-0..
+00305504: 2038 1100 0060 0324 0000 e290 0000 e490   8...`.$........
+00305514: ffff 6324 2620 8200 0000 e4a0 fbff 6104  ..c$& ........a.
+00305524: 0100 e724 d080 80af 2e01 2292 2f01 2392  ...$......"./.#.
+00305534: 2610 4300 1a00 0324 0300 4314 0100 0724  &.C....$..C....$
+00305544: bd05 0408 cc80 87af 0000 0724 bd05 0408  ...........$....
+00305554: cc80 87af 1000 0010 661e 4234 2d30 0000  ........f.B4-0..
 00305564: 2d28 2002 0000 a28c 0000 0000 0100 c624  -( ............$
 00305574: d080 838f 0400 a524 0018 c42c 2118 6200  .......$...,!.b.
 00305584: d080 83af f9ff 8054 0000 a28c d080 838f  .......T........
```

### SCPH-39004.rom1

❌ Different — `f3dfc19904e169ba1c77e6e28dc63acc` vs `a90245347069ce6bfb17886017e0f251`

- Total differing bytes: **89**

#### Region 1 — offset ~0xD4412 (869394 decimal)

```diff
--- diff-report/modchip-off_vs_modchip-ps2-mode_SCPH-39004.rom1_region1_A.hex	2026-07-05 22:43:30.024067201 +0200
+++ diff-report/modchip-off_vs_modchip-ps2-mode_SCPH-39004.rom1_region1_B.hex	2026-07-05 22:43:30.025034972 +0200
@@ -2,18 +2,18 @@
 000d4422: 5f8e 44d1 66fe 851c def6 e63e 24d6 f681  _.D.f......>$...
 000d4432: db16 8551 f757 5d4a 5dd2 49d6 4f37 a12f  ...Q.W]J].I.O7./
 000d4442: 155b f1f1 5f61 012d 11cc 6fe4 bbf4 0ce2  .[.._a.-..o.....
-000d4452: e7bd c5ed 1e98 1828 9166 b069 a653 147d  .......(.f.i.S.}
-000d4462: 98ed eef4 6c73 93a6 1a50 19c5 db1a f721  ....ls...P.....!
-000d4472: cf6a c7e0 8d4e 5be2 7d7c 04fd ef0f 32a9  .j...N[.}|....2.
-000d4482: 7c21 bd00 5151 9fa1 5c11 f23d 3af1 4655  |!..QQ..\..=:.FU
-000d4492: e727 b5d4 48a2 8096 7a45 b4f2 4fc5 13a9  .'..H...zE..O...
-000d44a2: b5d8 146a a76b 35a8 22bb f615 b596 b2ee  ...j.k5.".......
-000d44b2: d228 8ff9 db2d 8547 8548 91f9 12fe b7f1  .(...-.G.H......
-000d44c2: fef1 42aa 15f0 b4f5 894d b3fc 8d62 c01b  ..B......M...b..
-000d44d2: 30d5 ecf5 8f10 1c15 deeb c687 92f3 afc2  0...............
-000d44e2: 6684 58c2 4dc2 6e34 66fb 2e06 5ff6 ceb8  f.X.M.n4f..._...
-000d44f2: 9a78 2dae 4d2f 3fc5 6642 754e ea4a fca9  .x-.M/?.fBuN.J..
-000d4502: 08d7 f29c a2c9 3df1 5044 0ec7 48c9 6590  ......=.PD..H.e.
+000d4452: 00bd 00ed 0098 0028 0066 2069 3853 117d  .......(.f i8S.}
+000d4462: 00ed 00f4 6073 03a6 2450 00c5 001a e221  ....`s..$P.....!
+000d4472: 906a 00e0 004e e4e2 907c fffd ff0f 63a9  .j...N...|....c.
+000d4482: 2421 2600 2051 82a1 0011 003d 00f1 e455  $!&. Q.....=...U
+000d4492: a027 fbd4 ffa2 6196 0445 01f2 00c5 e7a9  .'....a..E......
+000d44a2: 24d8 d06a 806b 80a8 afbb 2e15 0196 22ee  $..j.k........".
+000d44b2: 9228 2ff9 012d 2347 9248 26f9 10fe 43f1  .(/..-#G.H&...C.
+000d44c2: 00f1 1aaa 00f0 03f5 244d 03fc 0062 431b  ........$M...bC.
+000d44d2: 14d5 01f5 0010 0715 24eb bd87 05f3 04c2  ........$.......
+000d44e2: 0884 ccc2 80c2 8734 affb 0006 00f6 07b8  .......4........
+000d44f2: 2478 bdae 052f 04c5 0842 cc4e 804a 87a9  $x.../...B.N.J..
+000d4502: afd7 f29c a2c9 3df1 5044 0ec7 48c9 6590  ......=.PD..H.e.
 000d4512: 4edf fd82 f596 3940 8df2 4902 4bd5 2d6d  N.....9@..I.K.-m
 000d4522: b0f1 f7f6 f21e 4d10 7bd9 9ede e286 fb86  ......M.{.......
 000d4532: b125 85d7 7158 40eb 8922 309e 69b9 5ebb  .%..qX@.."0.i.^.
```

### SCPH-39004.rom2

✅ Identical — MD5 `408579dfa40d6f9a0bd87cf1d8bc6a34`

## 🔹 modchip-off vs modchip-psx-mode

### SCPH-39004.mec

✅ Identical — MD5 `50500a02816976628774affb8632c6cc`

### SCPH-39004.nvm

✅ Identical — MD5 `5ed66bcef0550b55bc91d0afde6f728a`

### SCPH-39004.rom0

❌ Different — `0d2228e6fd4fb639c9c39d077a9ec10c` vs `49ae2b543d6709424dc4413f8a83f0ba`

- Total differing bytes: **1**

#### Region 1 — offset ~0x7FF12 (524050 decimal)

```diff
--- diff-report/modchip-off_vs_modchip-psx-mode_SCPH-39004.rom0_region1_A.hex	2026-07-05 22:43:30.070069643 +0200
+++ diff-report/modchip-off_vs_modchip-psx-mode_SCPH-39004.rom0_region1_B.hex	2026-07-05 22:43:30.071069696 +0200
@@ -2,7 +2,7 @@
 0007ff22: 0000 0000 0000 0000 0000 0000 0000 0000  ................
 0007ff32: 5379 7374 656d 2052 4f4d 2056 6572 7369  System ROM Versi
 0007ff42: 6f6e 2035 2e30 2030 332f 3139 2f30 3220  on 5.0 03/19/02 
-0007ff52: 4500 436f 7079 7269 6768 7420 3139 3933  E.Copyright 1993
+0007ff52: 4100 436f 7079 7269 6768 7420 3139 3933  A.Copyright 1993
 0007ff62: 2d31 3939 3920 2843 2920 536f 6e79 2043  -1999 (C) Sony C
 0007ff72: 6f6d 7075 7465 7220 456e 7465 7274 6169  omputer Entertai
 0007ff82: 6e6d 656e 7420 496e 632e 0000 0000 0000  nment Inc.......
```

### SCPH-39004.rom1

✅ Identical — MD5 `f3dfc19904e169ba1c77e6e28dc63acc`

### SCPH-39004.rom2

✅ Identical — MD5 `408579dfa40d6f9a0bd87cf1d8bc6a34`

## 🔹 modchip-ps2-mode vs modchip-psx-mode

### SCPH-39004.mec

✅ Identical — MD5 `50500a02816976628774affb8632c6cc`

### SCPH-39004.nvm

✅ Identical — MD5 `5ed66bcef0550b55bc91d0afde6f728a`

### SCPH-39004.rom0

❌ Different — `cc9f165880b52706c00abe0cdb1d6d4b` vs `49ae2b543d6709424dc4413f8a83f0ba`

- Total differing bytes: **76**

#### Region 1 — offset ~0x7FF12 (524050 decimal)

```diff
--- diff-report/modchip-ps2-mode_vs_modchip-psx-mode_SCPH-39004.rom0_region1_A.hex	2026-07-05 22:43:30.133635492 +0200
+++ diff-report/modchip-ps2-mode_vs_modchip-psx-mode_SCPH-39004.rom0_region1_B.hex	2026-07-05 22:43:30.134073041 +0200
@@ -2,7 +2,7 @@
 0007ff22: 0000 0000 0000 0000 0000 0000 0000 0000  ................
 0007ff32: 5379 7374 656d 2052 4f4d 2056 6572 7369  System ROM Versi
 0007ff42: 6f6e 2035 2e30 2030 332f 3139 2f30 3220  on 5.0 03/19/02 
-0007ff52: 4500 436f 7079 7269 6768 7420 3139 3933  E.Copyright 1993
+0007ff52: 4100 436f 7079 7269 6768 7420 3139 3933  A.Copyright 1993
 0007ff62: 2d31 3939 3920 2843 2920 536f 6e79 2043  -1999 (C) Sony C
 0007ff72: 6f6d 7075 7465 7220 456e 7465 7274 6169  omputer Entertai
 0007ff82: 6e6d 656e 7420 496e 632e 0000 0000 0000  nment Inc.......
```

#### Region 2 — offset ~0x3054C4 (3167428 decimal)

```diff
--- diff-report/modchip-ps2-mode_vs_modchip-psx-mode_SCPH-39004.rom0_region2_A.hex	2026-07-05 22:43:30.137972568 +0200
+++ diff-report/modchip-ps2-mode_vs_modchip-psx-mode_SCPH-39004.rom0_region2_B.hex	2026-07-05 22:43:30.138916006 +0200
@@ -2,12 +2,12 @@
 003054d4: 0000 0000 0000 0000 faff 6014 ffff 6324  ..........`...c$
 003054e4: f4ff 4014 2d20 0000 2d28 0000 202d 040c  ..@.- ..-(.. -..
 003054f4: 2d30 0000 d080 80af 1e08 040c 0000 0000  -0..............
-00305504: 2038 1100 0060 0324 0000 e290 0000 e490   8...`.$........
-00305514: ffff 6324 2620 8200 0000 e4a0 fbff 6104  ..c$& ........a.
-00305524: 0100 e724 d080 80af 2e01 2292 2f01 2392  ...$......"./.#.
-00305534: 2610 4300 1a00 0324 0300 4314 0100 0724  &.C....$..C....$
-00305544: bd05 0408 cc80 87af 0000 0724 bd05 0408  ...........$....
-00305554: cc80 87af 1000 0010 661e 4234 2d30 0000  ........f.B4-0..
+00305504: 2d18 4000 0500 6010 0200 0224 1300 6210  -.@...`....$..b.
+00305514: cc80 878f 2200 0010 0000 0000 cc80 878f  ...."...........
+00305524: 2d30 0000 2d28 2002 0000 a28c 0100 c624  -0..-( ........$
+00305534: d080 838f 0400 a524 0018 c42c 2118 6200  .......$...,!.b.
+00305544: d080 83af f9ff 8054 0000 a28c d080 838f  .......T........
+00305554: db62 023c 1000 0010 661e 4234 2d30 0000  .b.<....f.B4-0..
 00305564: 2d28 2002 0000 a28c 0000 0000 0100 c624  -( ............$
 00305574: d080 838f 0400 a524 0018 c42c 2118 6200  .......$...,!.b.
 00305584: d080 83af f9ff 8054 0000 a28c d080 838f  .......T........
```

### SCPH-39004.rom1

❌ Different — `a90245347069ce6bfb17886017e0f251` vs `f3dfc19904e169ba1c77e6e28dc63acc`

- Total differing bytes: **89**

#### Region 1 — offset ~0xD4412 (869394 decimal)

```diff
--- diff-report/modchip-ps2-mode_vs_modchip-psx-mode_SCPH-39004.rom1_region1_A.hex	2026-07-05 22:43:30.166023022 +0200
+++ diff-report/modchip-ps2-mode_vs_modchip-psx-mode_SCPH-39004.rom1_region1_B.hex	2026-07-05 22:43:30.166751485 +0200
@@ -2,18 +2,18 @@
 000d4422: 5f8e 44d1 66fe 851c def6 e63e 24d6 f681  _.D.f......>$...
 000d4432: db16 8551 f757 5d4a 5dd2 49d6 4f37 a12f  ...Q.W]J].I.O7./
 000d4442: 155b f1f1 5f61 012d 11cc 6fe4 bbf4 0ce2  .[.._a.-..o.....
-000d4452: 00bd 00ed 0098 0028 0066 2069 3853 117d  .......(.f i8S.}
-000d4462: 00ed 00f4 6073 03a6 2450 00c5 001a e221  ....`s..$P.....!
-000d4472: 906a 00e0 004e e4e2 907c fffd ff0f 63a9  .j...N...|....c.
-000d4482: 2421 2600 2051 82a1 0011 003d 00f1 e455  $!&. Q.....=...U
-000d4492: a027 fbd4 ffa2 6196 0445 01f2 00c5 e7a9  .'....a..E......
-000d44a2: 24d8 d06a 806b 80a8 afbb 2e15 0196 22ee  $..j.k........".
-000d44b2: 9228 2ff9 012d 2347 9248 26f9 10fe 43f1  .(/..-#G.H&...C.
-000d44c2: 00f1 1aaa 00f0 03f5 244d 03fc 0062 431b  ........$M...bC.
-000d44d2: 14d5 01f5 0010 0715 24eb bd87 05f3 04c2  ........$.......
-000d44e2: 0884 ccc2 80c2 8734 affb 0006 00f6 07b8  .......4........
-000d44f2: 2478 bdae 052f 04c5 0842 cc4e 804a 87a9  $x.../...B.N.J..
-000d4502: afd7 f29c a2c9 3df1 5044 0ec7 48c9 6590  ......=.PD..H.e.
+000d4452: e7bd c5ed 1e98 1828 9166 b069 a653 147d  .......(.f.i.S.}
+000d4462: 98ed eef4 6c73 93a6 1a50 19c5 db1a f721  ....ls...P.....!
+000d4472: cf6a c7e0 8d4e 5be2 7d7c 04fd ef0f 32a9  .j...N[.}|....2.
+000d4482: 7c21 bd00 5151 9fa1 5c11 f23d 3af1 4655  |!..QQ..\..=:.FU
+000d4492: e727 b5d4 48a2 8096 7a45 b4f2 4fc5 13a9  .'..H...zE..O...
+000d44a2: b5d8 146a a76b 35a8 22bb f615 b596 b2ee  ...j.k5.".......
+000d44b2: d228 8ff9 db2d 8547 8548 91f9 12fe b7f1  .(...-.G.H......
+000d44c2: fef1 42aa 15f0 b4f5 894d b3fc 8d62 c01b  ..B......M...b..
+000d44d2: 30d5 ecf5 8f10 1c15 deeb c687 92f3 afc2  0...............
+000d44e2: 6684 58c2 4dc2 6e34 66fb 2e06 5ff6 ceb8  f.X.M.n4f..._...
+000d44f2: 9a78 2dae 4d2f 3fc5 6642 754e ea4a fca9  .x-.M/?.fBuN.J..
+000d4502: 08d7 f29c a2c9 3df1 5044 0ec7 48c9 6590  ......=.PD..H.e.
 000d4512: 4edf fd82 f596 3940 8df2 4902 4bd5 2d6d  N.....9@..I.K.-m
 000d4522: b0f1 f7f6 f21e 4d10 7bd9 9ede e286 fb86  ......M.{.......
 000d4532: b125 85d7 7158 40eb 8922 309e 69b9 5ebb  .%..qX@.."0.i.^.
```

### SCPH-39004.rom2

✅ Identical — MD5 `408579dfa40d6f9a0bd87cf1d8bc6a34`

**Comment by neoconzer** [comment]
Excuse me, I'd like to chime in. The Redump database contains original BIOS Datfile checksums for all models of PS2.
First, download the latest version of PS2Ident. Then, use the PS2Ident to dump all files, with the modchip enabled and disabled respectively, compare the dumped files with the original data file.
Normally, only boot rom's checksum differs from the original boot rom. However, if the checksum of the boot rom after disabling the modchip is the same as the original, then it proves that the modchip can be truly disabled. 

http://redump.org/datfile/ps2-bios/
https://github.com/ps2homebrew/PS2Ident/releases

Hopefully, this will provide some assistance.

**Comment by PhilRoll** [comment]
> Excuse me, I'd like to chime in. The Redump database contains original BIOS Datfile checksums for all models of PS2. First, download the latest version of PS2Ident. Then, use the PS2Ident to dump all files, with the modchip enabled and disabled respectively, compare the dumped files with the original data file. Normally, only boot rom's checksum differs from the original boot rom. However, if the checksum of the boot rom after disabling the modchip is the same as the original, then it proves that the modchip can be truly disabled.
> 
> http://redump.org/datfile/ps2-bios/ https://github.com/ps2homebrew/PS2Ident/releases
> 
> Hopefully, this will provide some assistance.

Thank you so much for the tip!!

I redid the dump with the modchip disabled using PS2Ident, then compared it against the redump BIOS datfile — both the BOOT_ROM and DVD_ROM match perfectly (dunno about DVD_ROM and NVM files):

File | My MD5 | DAT MD5 | Match
-- | -- | -- | --
SCPH-39004_BOOT_ROM.bin | 0d2228e6fd4fb639c9c39d077a9ec10c | 0d2228e6fd4fb639c9c39d077a9ec10c | ✅
SCPH-39004_database.bin | 8af88163cf913d631a9e83fb1125b430 | missing | ❌
SCPH-39004_DVD_ROM.bin | 6bdb45a952f697f367cd1646cdf09235 | 6bdb45a952f697f367cd1646cdf09235 | ✅
SCPH-39004_NVM.bin | 5ed66bcef0550b55bc91d0afde6f728a | missing | ❌



For reference, here's the full PS2Ident log confirming BIOS version 5.0 (03/19/02, region E) on SCPH-39004:
<pre><code>Log file generated by Playstation 2 Ident v0.850, built on Jun  3 2022 07:06:51

ROMVER:            0160EC20020319
ROM region sizes:
    ROM0:          0xBFC00000 (3887451 bytes)
    ROM1:          0xBE000000 (199099 bytes)
    ROM2:          &lt;Not detected&gt;
    EROM:          0xBE040000 (1835008 bytes)
ROM chip sizes:
    Boot ROM:      0x1FC00000 (32 Mbit)    CRC16: 0xa9f7
    DVD ROM:       0x1E000000 (16 Mbit)    CRC16: 0x8df1
    DVD Player:    2.13E
    OSDVer:        -
    PS1DRV:        1.10
EE/GS:
    Implementation:      0x2e
    Revision:            3.1 (CXD9708GB/CXD9832GB)
    EE_F520:             0x00003101
    EE_F540:             0x008ce8c2
    EE_F550:             0x0090c808
    FPU implementation:  0x2e
    FPU revision:        3.0
    ICache size:         0x02 (16 KB)
    DCache size:         0x01 (8 KB)
    RAM size:            33554432 bytes
    GS revision:         1.09 (CXD2949BGB)
    GS ID:               0x55
IOP:
    Implementation:      0x00
    Revision:            2.1 (CXD9697GP/CXD9732GP)
    RAM size:            2097152 bytes
    SSBUS I/F revision:  3.1 (CXD9611)
    AIF revision:        &lt;Not detected&gt;
MECHACON:
    Revision:            3.06 (CXP103049-002GG/-(1,2,3)03GG/-402GG/-501GG)
    MagicGate region:    0x02 (Europe)
    System type:         0x00 (PlayStation 2)
    DSP revision:        3 (CXD3098Q/CXD1886Q-1/CXD3098AQ)
    M Renewal Date:      ----/--/-- --:--
Mainboard:
    Model name:          SCPH-39004
    Mainboard model:     Missing
    Chassis:             Unknown
    ROMGEN:              0319-2002
    Machine type:        0x00000000
    BoardInf:            0x00 (Not detected)
    MPU Board ID:        0xffff
    SPU2 revision:       0x08 (CXD2947R/CXD2950R)
    ADD0x010:            0xb029 (G SANYO)
    i.Link Model ID:     0x1a001c (SCPH-39004)
    Console Model ID:    0x1b
    SDMI Company ID:     00-00-d2
    EMCS ID:             0x01 (FOXC)
    Serial range:        372xxxx
    USB HC revision:     1.0
DEV9:
    MAC vendor:          00:04:1f
    SPEED revision:      0x0013 (CXD9731GP)
    SPEED capabilities:  000b.0002 (SMAP, ATA, UART)
    PHY OUI:             0x080017 (National Semiconductor)
    PHY model:           0x02 (DP83846A)
    PHY revision:        0x03
i.Link:
    Ports:               2
    Max speed:           2 (S400)
    Compliance level:    1 (IEEE1394A-2000)
    Vendor ID:           0x00a0b8 (LSI Logic)
    Product ID:          0x600201
</code></pre>

**Comment by neoconzer** [comment]
According to the results, I think the modchip should be truly disabled.
NVM should be related to optical drives and mechacon, you can compare the two dumped files after disabling the modchip and enabling the modchip. They should be the same, but I am not sure whether this is related to the problem you encountered. Need to wait for the developer to check it.

**Comment by DKWDRV** [comment]
@PhilRoll does the disc ever spin?
Try the new version 1.7.6L

**Comment by DKWDRV** [comment]
@PhilRoll you also have 1.7.6m to try.

**Comment by PhilRoll** [comment]
@DKWDRV tested 1.7.6.L and 1.7.6.m, nothing :(

No matter what combination I tried, the disc wouldn't spin.
I also tried using the RCA video output, but nothing happen...

PS: A general question:
Is it normal that, when i run DKWDRV, the hard drive to start running if the PS2 network adapter is plugged? (I can tell because the fan speeds up.)
When I press “RUN DISC,” the fan noise goes back to normal.

**Comment by josselcembrano-star** [comment]
Hi everyone, for reference, i have a modchip ps2 fat scph 3000. I boot it by pressing circle to make it in ps1 mode so it doesnt read ps1 backup disc as ps2. I run dkwdrv, scan disc, make sure to sselect the component fix to progressive scan WIP (setting it only to yes gives me invalid format in my tv). Select run disc and it works.

I used legend of mana backup disc.

**Comment by DKWDRV** [comment]
@PhilRoll try 1.7.6n

**Comment by DKWDRV** [comment]
> [@DKWDRV](https://github.com/DKWDRV) tested 1.7.6.L and 1.7.6.m, nothing :(
> 
> No matter what combination I tried, the disc wouldn't spin. I also tried using the RCA video output, but nothing happen...
> 
> PS: A general question: Is it normal that, when i run DKWDRV, the hard drive to start running if the PS2 network adapter is plugged? (I can tell because the fan speeds up.) When I press “RUN DISC,” the fan noise goes back to normal.

It loads dev9 on bootup and also turns off dev9 before launching game so I guess it is normal.

**Comment by DKWDRV** [comment]
@PhilRoll unlikely but try 1.7.6o too.

**Comment by DKWDRV** [comment]
> > [@PhilRoll](https://github.com/PhilRoll) can you try with different Component fix options? Why not unpack your two bios dump chips and tell us which file of the rom changes?
> 
> [@DKWDRV](https://github.com/DKWDRV) Yes, i tried component/ps2hdmi fix disabled, enabled and progressive mode (wip). The problem is that the disc isn't spinning at all. “Scan Disc” works, and I can see the data correctly under “Info”. But as soon as I press “RUN DISC,” the screen goes black (or there's no video output if I disable the component fix) And, most important thing, the disc isn't spinning, in any video mode after pressing run.
> 
> I dumped and unpacked the BIOS for all **3 states** of the modchip (Matrix MXL2):
> 
>     * Chipmod **OFF**
> 
>     * Chipmod **ON – PS2 mode**
> 
>     * Chipmod **ON – PSX mode**
> 
> 
> Console: **SCPH-39004**
> File 	OFF 	PS2 mode 	PSX mode 	Result
> rom0 	`0d2228e6fd4fb639c9c39d077a9ec10c` 	`cc9f165880b52706c00abe0cdb1d6d4b` 	`49ae2b543d6709424dc4413f8a83f0ba` 	❌ all different
> rom1 	`f3dfc19904e169ba1c77e6e28dc63acc` 	`a90245347069ce6bfb17886017e0f251` 	`f3dfc19904e169ba1c77e6e28dc63acc` 	❌ PS2 mode differs
> rom2 	`408579dfa40d6f9a0bd87cf1d8bc6a34` 	same 	same 	✅ identical
> nvm 	`5ed66bcef0550b55bc91d0afde6f728a` 	same 	same 	✅ identical
> mec 	`50500a02816976628774affb8632c6cc` 	same 	same 	✅ identical
> 
> Even though `rom0` and `rom1` differ as whole files, unpacking each ROM into its individual modules shows the actual changes are minimal:
> 
> **rom0** (92 modules) — only **2 modules** differ:
> Module 	# 	Differs in 	Notes
> `ROMGSCRT` 	54 	**PSX mode** only 	OFF and PS2 mode are identical to each other
> `PS2LOGO` 	88 	**PS2 mode** only 	OFF and PSX mode are identical to each other
> 
> All other 90 modules in rom0 are byte-identical across all 3 states.
> 
> **rom1** (15 modules) — **0 modules** differ, despite the file MD5 changing in PS2 mode. That means the difference lives in a region of rom1 that isn't covered by the module table (likely boot/config data specific to the modchip's IOP patching, not extracted as a named module).
> 
> **rom2, nvm, mec** — fully identical in all 3 states, both at file and module level.

@PhilRoll you show here that ROMGSCRT changes with PSX mode but in later report with bytes no byte of it seems to impacting that file. Can you show the byte changes of that file in PSX mode?

**Comment by PhilRoll** [comment]
@DKWDRV 1.7.6o doesn't work..



## File checksums

| File | MD5 | Size (bytes) |
|------|-----|---------------|
| SCPH-39004.mec | `50500a02816976628774affb8632c6cc` | 4 |
| SCPH-39004.nvm | `5ed66bcef0550b55bc91d0afde6f728a` | 1024 |
| SCPH-39004.rom0 | `49ae2b543d6709424dc4413f8a83f0ba` | 4194304 |
| SCPH-39004.rom1 | `f3dfc19904e169ba1c77e6e28dc63acc` | 4194304 |
| SCPH-39004.rom2 | `408579dfa40d6f9a0bd87cf1d8bc6a34` | 524288 |

---

## Byte-level diff vs `modchip-off` baseline


### SCPH-39004.mec

✅ Identical to baseline — MD5 `50500a02816976628774affb8632c6cc`

### SCPH-39004.nvm

✅ Identical to baseline — MD5 `5ed66bcef0550b55bc91d0afde6f728a`

### SCPH-39004.rom0

❌ Differs from baseline — `0d2228e6fd4fb639c9c39d077a9ec10c` (off) vs `49ae2b543d6709424dc4413f8a83f0ba` (modchip-psx-mode)

- Total differing bytes: **1**

#### Region 1 — offset ~0x7FF12 (524050 decimal)

```diff
@@ -2,7 +2,7 @@
 0007ff22: 0000 0000 0000 0000 0000 0000 0000 0000  ................
 0007ff32: 5379 7374 656d 2052 4f4d 2056 6572 7369  System ROM Versi
 0007ff42: 6f6e 2035 2e30 2030 332f 3139 2f30 3220  on 5.0 03/19/02 
-0007ff52: 4500 436f 7079 7269 6768 7420 3139 3933  E.Copyright 1993
+0007ff52: 4100 436f 7079 7269 6768 7420 3139 3933  A.Copyright 1993
 0007ff62: 2d31 3939 3920 2843 2920 536f 6e79 2043  -1999 (C) Sony C
 0007ff72: 6f6d 7075 7465 7220 456e 7465 7274 6169  omputer Entertai
 0007ff82: 6e6d 656e 7420 496e 632e 0000 0000 0000  nment Inc.......
```

### SCPH-39004.rom1

✅ Identical to baseline — MD5 `f3dfc19904e169ba1c77e6e28dc63acc`

### SCPH-39004.rom2

✅ Identical to baseline — MD5 `408579dfa40d6f9a0bd87cf1d8bc6a34`

**Comment by neoconzer** [comment]
> PS: A general question: Is it normal that, when i run DKWDRV, the hard drive to start running if the PS2 network adapter is plugged? (I can tell because the fan speeds up.) When I press “RUN DISC,” the fan noise goes back to normal.

I'd like to provide some information about my ps2 for your reference.

Whenever my ps2 is powered on with a hard drive connected, the fan automatically runs at high speed, regardless of whether the hard drive is actually running, until the ps2 is shut down.

It seems the DKWDRV doesn't access the internal HDD, at least I'm certain that the hard drive doesn't boot whenever I run the DKWDRV.

**Comment by DKWDRV** [comment]
@PhilRoll does the disc ever spin? That would be the first sign of progress?
I really do not have any more ideas about it. It only leaves your modchip.

**Comment by neoconzer** [comment]
I noticed that the PS2Ident logs showed that it did not detect the Mainboard model and Chassis. I'm unsure if this is related.

**Comment by PhilRoll** [comment]
@DKWDRV no the disc doesn't spin, or rather: when I press run disc I feel the lens moving for a micro second and the disc stopping (all this in the very first second).

**Comment by DKWDRV** [comment]
Then most likely something is failing. Perhaps the chip waits some specific sequence from the PS1DRV to do something when the IOP switches to PS1 mode.

**Comment by neoconzer** [comment]
I think you can try this method: Run version 1.7.2. Don't insert the game disc yet. Enable HDMI Fix in the settings. Then insert the disc, wait a second or two for the ps2 to recognize the game disc, then directly select "RUN." Some warning messages will appear on the screen, ignore them and still press X to continue anyway. See if the game launches.

Some additional tips: If a black screen appears after running 1.7.2, please keep the disc tray open while running the ELF.

https://github.com/wisi-w/DKWDRV/releases

**Comment by PhilRoll** [comment]
> I think you can try this method: Run version 1.7.2. Don't insert the game disc yet. Enable HDMI Fix in the settings. Then insert the disc, wait a second or two for the ps2 to recognize the game disc, then directly select "RUN." Some warning messages will appear on the screen, ignore them and still press X to continue anyway. See if the game launches.
> 
> Some additional tips: If a black screen appears after running 1.7.2, please keep the disc tray open while running the ELF.
> 
> https://github.com/wisi-w/DKWDRV/releases

@DKWDRV I tried version 1.7.2 following your exact steps and it worked perfectly!

Here is what I did:
1. Launched the 1.7.2 ELF without the disc inside.
2. Enabled the HDMI Fix in the settings.
3. Inserted the disc, waited a couple of seconds for the PS2 to recognize it, and hit RUN.
4. Ignored the warning messages and pressed X to bypass them.

The game booted!!!!

**Comment by DKWDRV** [comment]
@PhilRoll @neoconzer will anyone care the explain this black magic?

**Comment by neoconzer** [comment]
Hi @PhilRoll! Could you test and confirm again that if 1.7.2 is run and no disc is inserted in the optical drive and the tray is kept closed, will the screen go black?

**Comment by neoconzer** [comment]
> [@neoconzer](https://github.com/neoconzer) will anyone care the explain this black magic?

To be honest, I don't know the underlying principle either. The reason I had him try 1.7.2 was for two reasons: first, 1.7.2 seemed to have fewer changes, or rather, it retained more of the original driver's characteristics, second, in my memory, 1.7.2 wouldn't stop spinning if the disc wasn't scanned (if you insert a disc before running 1.7.2, it will automatically scans discs and has no options), unlike later versions where the disc would automatically stop spinning once it was inserted into the drive, even without scanning. 

However, I'm not sure if this is related to the problem PhilRoll encountered. But 1.7.2 eventually launched the game successfully, indicating that there must be some reason behind it.

**Comment by DKWDRV** [comment]
Maybe it's some type of modchip hook thing once again.
@PhilRoll needs to more more tests to figure the quirk behind this.

**Comment by PhilRoll** [comment]
> Hi [@PhilRoll](https://github.com/PhilRoll)! Could you test and confirm again that if 1.7.2 is run and no disc is inserted in the optical drive and the tray is kept closed, will the screen go black?

I just tried running the version 1.7.2 without any disc inserted and with the tray closed. When I selected "RUN", the screen went completely black.

@DKWDRV What other tests could i try?

**Comment by DKWDRV** [comment]
@PhilRoll try different versions going to Scan Disc and scanning/skipping then opening/closing tray, waiting for it to spin or anything and then going to RUN

**Comment by PhilRoll** [comment]
@DKWDRV good news:

I started the test run with version 1.7.6o

I performed this exact same test both with the modchip disabled and with the modchip in PS1 mode:

1. I launched DKWDRV 1.7.6o with the disc tray open

2. I closed the disc tray; at the exact moment the tray closed, I counted 4 seconds

3. At the exact fourth second, I performed a “scan disc”

4. I pressed “RUN”

5. Sometimes (not always) this warning message appears:
“No valid PS1 disc found!
You will probably get a black screen!
Unless it’s some modchip magic.
Make sure to insert a valid disc BEFORE launching.
Insert a valid disc and reset the console.
Launch this program again.
Verify in the INFO tab.”
I pressed X to continue anyway

6. This warning message appears:
“This console enforces strict disc checking! The inserted disc does not pass the logo checks! The license string/data is incorrect or for a different region! Booting the game will result in a black screen! Only consoles with modchips and PS1 patching will still work! Press X to continue anyway.”
I pressed X to continue anyway

The game launched!

Tomorrow, as soon as I can, I’ll run more tests with other versions as well.

What I’m realizing is that, crucially, the timing with which I close the tray and press “Scan Disc” is key to getting the disc to start.

I’ve also tried launching the ELF with the disc inserted, so I open and close the disc tray again, sticking to the timing: “the moment the tray closes, I count exactly 4 seconds and press ‘Scan Disc.’” 
Infact, if I wait 5+ seconds after inserting the disc and then run “Scan Disc,” the game won’t start.

Step 5 doesn’t always appear; it really depends on how precisely I follow this timing.

**Comment by DKWDRV** [comment]
After you insert the disc you can check in INFO menu the disc type. That value will change once mecha parses the disc.

After you click Scan Disc and RUN and game boots, in such case after the Scan Disc did the disc stop and only spins once on RUN?

**Comment by neoconzer** [comment]
Please also test this by inserting the game disc before launching 1.7.2, and then selecting "Run" to see if the game still launches.

Based on the results, I think we can determine whether this is related to keeping the disc spinning continuously.

**Comment by PhilRoll** [comment]
@neoconzer 

1.7.6.o:
Following the same technique of keeping the disc tray open, closing it, and waiting exactly 4 seconds:
the moment I press "scan disc," the disc spins, then stops.
Sometimes the disc is recognized immediately; other times I have to perform a second scan to get the correct INFO. 
As I mentioned, it depends on how long I wait before pressing “scan disc.”
In any case, when I press RUN (and any warnings appear), pressing X the disc to start spinning and the game start.


1.7.2
If I leave the disc inside, launch the app, and then press RUN, the screen goes black and the disc isn’t spinning.
It only works if I use the technique of opening and closing the disc tray.

I'll do some more confirmatory tests later, but I can't right now.

**Comment by neoconzer** [comment]
The result is that if the disc is inserted beforehand and then 1.7.2 is launched, the game cannot be launched, and the disc does not spin. Based on my previous description of the features of 1.7.2, I personally believe that the problem is very likely related to whether the disc can be kept spinning continuously.

For 1.7.6, I think you can try not scanning the disc, but instead wait 3 to 4 seconds and then directly select "Run," then press the "X" button repeatedly. I guess there's a high chance the game will launch.

**Comment by DKWDRV** [comment]
Possible related to [this](https://github.com/pcm720/OSDMenu/blob/adcdecf56b3c4a8838b1e778f5e128e0132e1697/launcher/src/handler_cdrom.c#L357). @pcm720

**Comment by pcm720** [comment]
Current PS2SDK changes for some reason broke this for all the other models, not just PGIFs...
I couldn't find a better workaround other than opening and closing the tray.

**Comment by DKWDRV** [comment]
> Current PS2SDK changes for some reason broke this for all the other models, not just PGIFs... I couldn't find a better workaround other than opening and closing the tray.

Which change? What exactly does this breaking? When loading an ELF or only OSDSYS?

**Comment by pcm720** [comment]
> Which change? What exactly does this breaking?

No idea.
It has been happening for a while on non-slim consoles.
Some of the June changes also broke this for slims as noted by @saildot4k

> When loading an ELF or only OSDSYS?

If you turn the console on with the disc already inserted and load a PS2SDK-based ELF from OSDSYS, the disc won't be recognized unless the tray is opened and closed. On slim consoles, only these two commands (tray open/close) are enough to make it spin the disc and recognize it.
Maybe it's a XCDVDMAN + SCE SDK -> CDVDMAN + PS2SDK libcdvd compatibility issue?
I haven't really looked into it

**Comment by DKWDRV** [comment]
> > Which change? What exactly does this breaking?
> 
> No idea. It has been happening for a while on non-slim consoles. Some of the June changes also broke this for slims as noted by [@saildot4k](https://github.com/saildot4k)
> 
> > When loading an ELF or only OSDSYS?
> 
> If you turn the console on with the disc already inserted and load a PS2SDK-based ELF from OSDSYS, the disc won't be recognized unless the tray is opened and closed. On slim consoles, only these two commands (tray open/close) are enough to make it spin the disc and recognize it. Maybe it's a XCDVDMAN + SCE SDK -> CDVDMAN + PS2SDK libcdvd compatibility issue? I haven't really looked into it

More details? Could very well be related to this issue somehow.

**Comment by pcm720** [comment]
> More details? Could very well be related to this issue somehow.

That's all the details I have.
If I knew what caused this, I'd fix it a long time ago

**Comment by DKWDRV** [comment]
> > More details? Could very well be related to this issue somehow.
> 
> That's all the details I have. If I knew what caused this, I'd fix it a long time ago

Does it fail to launch an ELF from cdrom0? Or sceCdInit/GetDiskType fails? Who reported this?

**Comment by pcm720** [comment]
> Does it fail to launch an ELF from cdrom0? 

Usually no, the game executable launches and hangs until the tray is cycled

> Or sceCdInit/GetDiskType fails?

No idea because the hang does not happen in my code.

> Who reported this?

No one. I've encountered it when testing disc launches, other people confirmed it also happened with the PSX.
Recent SDK changes breaking disc loading on Slim consoles were reported by @saildot4k

**Comment by neoconzer** [comment]
@DKWDRV  Maybe you can develop an internal test version first, just let the disc continue to spin regardless of whether it scans the disc after it is put into the disc compartment, and then let PhilRoll test it. If the problem can be fixed, then maybe only a disc control option needs to be added to the official version to let users choose by themselves.

**Comment by DKWDRV** [comment]
@PhilRoll  try 1.7.6p, this bug is a nightmare but several changes were made.

**Comment by PhilRoll** [comment]
@DKWDRV it works!!!!!

thank you so much for all the work and effort you put into solving this issue.

I tried with HDMI adapter and HDMI fix on, both disabling the modchip and with the modchip active in PS1 mode and it works in both cases!
(It doesn't work with the modchip in PS2 mode, but I guess that's normal).

**Comment by DKWDRV** [comment]
Thank you @PhilRoll for testing. Thank you @neoconzer @pcm720 @dskaufman1

**Comment by pcm720** [comment]
@DKWDRV 
Can you provide any actual details on how this was fixed?
"Spin the motor before the PS1 mode switch" is not very descriptive
